samples/EmbeddedTextContent.sarif

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "properties": {
+    "comment": "This sample contains a result in an artifact whose textual contents are embedded in the log file."
+  },
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SarifSamples"
+        }
+      },
+      "results": [
+        {
+          "ruleId": "SMP0001",
+          "message": {
+            "text": "The result."
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "hello.txt",
+                  "index": 0
+                },
+                "region": {
+                  "startLine": 2
+                }
+              }
+            }
+          ]
+        }
+      ],
+      "artifacts": [
+        {
+          "location": {
+            "uri": "hello.txt"
+          },
+          "contents": {
+            "text": "Hello,\r\nworld"
+          },
+          "encoding": "UTF-8"
```

Comment:
>encoding [](start = 11, length = 8)

this sample may be more interesting if you put a default encoding on the run, then provide two artifacts one of which receives the default, the other of which departs from it.

Reply:
Done